tests/ffi/dev-mem_not_present/test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+. ../../e2e/lib/utils
+
+check_dev-mem_not_present(){
+    # Check /dev/mem is not present in QM partition
+    if podman exec qm test -e /dev/mem; then
+        info_message "Found /dev/mem in QM partition: $(podman exec -t qm ls -l /dev/mem)"
+        info_message "FAIL: check_dev-mem_not_present: Check for /dev/mem in QM partition failed, it should not present."
```

Comment:
It should not be present

Reply:
@KittyMae Thank you for your careful review.
Done